tests/unittests/aggregation_tests.py

```
@@ -1127,7 +1128,7 @@ def test_object_ids(self):
     @drop_datasets
     def test_dates(self):
         today = date.today()
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)